x-pack/libbeat/management/manager_test.go

```diff
@@ -53,7 +53,7 @@ func TestConfigManager(t *testing.T) {
 		`{"configuration_blocks":[{"type":"test.block","config":{"module":"system"}}]}`,
 	}
 	mux.Handle("/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		assert.Equal(t, fmt.Sprintf("/api/beats/agent/%s/configuration", id), r.RequestURI)
+		assert.Equal(t, fmt.Sprintf("/api/beats/agent/%s/configuration?validSetting=true", id), r.RequestURI)
```

Comment:
@ruflin @exekias I know this is a cherry-pick, probably under time pressure, and already reviewed, but I'm worried about the test coverage here.

It seems that we only test the validSetting=true case for the manager. It really seems to me that we should check the validSetting=false case in the manager test.

We do a test where that value is false in configuration_test.go, but it needs to go through the manager abstraction to be useful.

Reply:
As this is a cherry pick, I'd opt for solving it in master and backport to 6.x and 6.5 branch later.